apps/io-profile/src/functions/send-templated-login-email-activity.ts

```diff
@@ -114,7 +114,7 @@ export const getSendLoginEmailActivityHandler =
         ),
       ),
       E.bind("emailText", ({ emailHtml }) =>
-        E.of(HtmlToText.fromString(emailHtml, emailDefaults.htmlToTextOptions)),
+        E.of(HtmlToText.htmlToText(emailHtml, emailDefaults.htmlToTextOptions)),
```

Comment:
We currently don't have any tests in place to catch regressions in this area. Can we improve [this](https://github.com/pagopa/io-auth-n-identity-domain/blob/0a4dc3f86031418849d18d8754a4dc9fdbaf67ed/apps/io-profile/src/functions/__tests__/send-templated-validation-email-activity.test.ts#L15) html content? I know that it cannot cover any scenario

Reply:
I added a dedicated test file in https://github.com/pagopa/io-auth-n-identity-domain/pull/256/commits/d3f64f655db0a101c94c85b50bd4ade63d19e896. let me know if it's what you were thinking